app/notifications/rest.py

```diff
@@ -320,3 +303,28 @@ def get_notification_statistics_for_day():
     )
     data, errors = notifications_statistics_schema.dump(statistics, many=True)
     return jsonify(data=data), 200
+
+
+def persist_notification(
+        service,
+        notification_id,
+        notification,
+        created_at,
+        notification_type,
+        api_key_id=None,
+        key_type=KEY_TYPE_NORMAL,
+):
+    dao_create_notification(
+        Notification.from_api_request(
+            created_at, notification, notification_id, service.id, notification_type, api_key_id, key_type
+        )
+    )
+
+    if notification_type == SMS_TYPE:
```

Comment:
We need to ensure that
1) DB write succeeds before SQS call is made.
2) If SQS call fails we need to ensure that the DB row is removed.
3) If SQS call fails we need to ensure a 500 response to the client.